src/syntax/FStar.Syntax.Util.fs

```diff
@@ -1708,7 +1708,7 @@ let rec term_eq_dbg (dbg : bool) t1 t2 =
     (check "app head"  (term_eq_dbg dbg f1 f2)) &&
     (check "app args"  (eqlist (arg_eq_dbg dbg) a1 a2))
 
-  | Tm_match (t1,bs1), Tm_match (t2,bs2) ->
+  | Tm_match (t1,None,bs1), Tm_match (t2,None,bs2) ->  //AR: note: no return annotations
```

Comment:
If we enhance eq_tm then we should also enhance this one to account for returns clauses?